go.mod

```diff
@@ -27,18 +31,21 @@ require (
 	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709 // indirect
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90 // indirect
 	github.com/prometheus/common v0.2.0 // indirect
 	github.com/prometheus/procfs v0.0.0-20190227231451-bbced9601137 // indirect
+	github.com/sirupsen/logrus v1.5.0 // indirect
```

Comment:
We are not using logrus. This can be removed. Also the checked in logrus files in vendor.

Reply:
Haven't imported it directly anywhere . It's being used by some other packages I imported so we cannot remove it.